README.md

```diff
@@ -88,7 +89,9 @@ Vagrant.configure("2") do |config|
     google.google_project_id = "YOUR_GOOGLE_CLOUD_PROJECT_ID"
     google.google_client_email = "YOUR_SERVICE_ACCOUNT_EMAIL_ADDRESS"
     google.google_json_key_location = "/path/to/your/private-key.json"
-
+    
+    google.image_family = 'ubuntu-1604-lts'
```

Comment:
One suggestion ... it would be nice if the new `image_family` config option was also added to the `Configuration` section of the README.

Reply:
@seanmalloy Nicely spotted! Thanks!

Added.